apps/hash-integration-worker/src/linear-activities.ts

```diff
@@ -132,10 +152,30 @@ const createHashEntity = async (params: {
       ],
     })),
   );
+
+  // TODO: allow creating policies alongside entity creation
+  //   see https://linear.app/hash/issue/H-4622/allow-creating-policies-alongside-entity-creation
+  for (const linkEntity of linkEntities) {
+    const linkEntityUuid = extractEntityUuidFromEntityId(linkEntity.entityId);
+    await createPolicy(graphApiClient, params.authentication, {
+      name: `linear-synced-administer-entity-${linkEntityUuid}`,
+      effect: "permit",
+      principal: {
+        type: "actor",
+        actorType: "machine",
+        id: params.authentication.actorId,
+      },
+      actions: ["viewEntity"],
```

Comment:
I believe this should be whatever the `administrator` equivalent is, as that's what we're adding on top of the web defaults above.

Reply:
Yup, that's currently captured under H-4442 (at the very bottom):
<img width="421" alt="image" src="https://github.com/user-attachments/assets/65716444-36f3-4bdb-8d16-cc7f41e5a0b0" />

I tried to somehow capture that to some policies more actions needs to be added in the future.